mart/attack/adversary.py

```diff
@@ -273,16 +263,17 @@ def forward(
         input: Union[torch.Tensor, tuple],
         target: Union[torch.Tensor, Dict[str, Any], tuple],
         model: Optional[torch.nn.Module] = None,
-        **kwargs
+        **kwargs,
```

Comment:
unnecessary (unless make style did this)

Reply:
Yes, it's enforced by pre-commit.